packages/compiler/src/server/dynamic-server-task.ts

```diff
@@ -0,0 +1,63 @@
+import { StatusIcons } from "../core/logger/dynamic-task.js";
+import { TaskStatus, TrackActionTask } from "../core/types.js";
+import { ServerLog } from "./types.js";
+
+export class DynamicServerTask implements TrackActionTask {
+  #log: (log: ServerLog) => void;
+  #message: string;
+  #interval: NodeJS.Timeout | undefined;
```

Comment:
do you need this interval? seems like this task tracker is not really dynamic?

Reply:
removed